chef_master/source/cookstyle.rst

```diff
@@ -9,18 +9,160 @@ Most of the code that is authored when working with Chef is written as Ruby. Jus
 
 .. end_tag
 
-cookstyle is a linting tool based on RuboCop designed to address the following issues:
+Cookstyle is a linting tool based on the RuboCop Ruby linting tool. Use Cookstyle to author better cookbook code:
 
-* New releases of RuboCop typically contain new style rules. These often cause projects to fail style checks after the upgrade.
-* The default configuration for RuboCop enables many checks that are inappropriate for cookbook development.
+* Enforce style conventions and best practices
+* Evaluate the code in a cookbook against metrics like "line length" and "function size"
```

Comment:
Let me track down some better metrics. This bugged me when I copied it over and I should have fixed it then.